config/custom-environment-variables.yaml

```diff
@@ -10,6 +10,8 @@ auth:
     # Generate one by running
     # $ openssl rsa -in jwt.pem -pubout -out jwt.pub
     jwtPublicKey: SECRET_JWT_PUBLIC_KEY
+    # Environment of the JWTs. For example: 'prod' or 'beta'
+    jwtEnvironment: JWT_ENVIRONMENT
```

Comment:
is this value the same as `PREFIX` we have for a bunch of settings ?

Reply:
No, it can be anything (`test`, `beta`, `prod`) depending on the cluster manager.

Reply:
makes sense, I was just checking if we already have any duplicate settings which captures this value already.